docs/scripts/buildApi.js

```diff
@@ -11,6 +10,7 @@ import { defaultHandlers, parse as docgenParse } from 'react-docgen';
 import remark from 'remark';
 import remarkVisit from 'unist-util-visit';
 import * as yargs from 'yargs';
+import { getLineFeed } from './helpers';
```

Comment:
nit: revert this re-arrangment for proper git history.

Reply:
This is included in one of the rules of the Airbnb preset

Reply:
Can we discuss introduction of new rules in another PR?

Reply:
Are you against the rule or the diff?

Reply:
If you are against the rule, happy to discuss it and revert if needed.

If you are against the diff, I think that we will have to pay it at some time anyway, you & I already paid the cost now. I'm not sure it's worth spending time on reversing it.

Reply:
Against the rule and diff.

Reply:
Ok, then, let's focus on the rule. I propose we run a vote, all in favor 👍, against 👎 and we settle with the majority. I don't think that it's very important. cc @mui-org/core-team. The rule in question is https://github.com/airbnb/javascript/blob/c5bee75b1b358a3749f1a6d38ee6fad73de28e29/packages/eslint-config-airbnb-base/rules/imports.js#L145-L148

Reply:
On this matter, we can also bring @dmtrKovalenko's extension to the table: https://github.com/dmtrKovalenko/eslint-plugin-pretty-imports.

Reply:
> I couldn't figure out the sorting logic behind the scene

What is the point of applying an order that is not reasonable to humans? I don't understand what ordering accomplishes here.

Reply:
Regarding ordering import, I think that the current rule communicates the broadness of the dependencies. *broadness* as the number of dependents. The first dependency being the one with the highest number of dependents: React. But in practice, it only split into two groups: global and relative imports. 

Why?

From my perspective it's about helping navigating the list, knowing it's sorted in a specific way.

Is it important?

I think that it's a net positive but at the end of the day, this alone won't make a difference.

Reply:
> From my perspective it's about helping navigating the list, knowing it's sorted in a specific way.

I think you don't know how it is sorted?

Reply:
I'm advocating against https://github.com/dmtrKovalenko/eslint-plugin-pretty-imports but in favor of Airbnb [default rule](https://github.com/airbnb/javascript/blob/c5bee75b1b358a3749f1a6d38ee6fad73de28e29/packages/eslint-config-airbnb-base/rules/imports.js#L145-L148).